automatic/screencloud/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,26 @@
+$ErrorActionPreference = 'Stop'
+
+[array]$key = Get-UninstallRegistryKey -SoftwareName 'ScreenCloud'
+
+if($key.Count -eq 1){
+    $key | % {
+        $file = "$($_.UninstallString)"
+
+        $packageArgs = @{
+            packageName    = 'screencloud'
+            fileType       = 'msi'
+            silentArgs     = "$($_.PSChildName) /qn /norestart"
+            validExitCodes = @(0)
+            file           = ''
+        }
+
+        Uninstall-ChocolateyPackage @packageArgs
+    }
+} elseif ($key.Count -eq 0){
+    Write-Warning "$packageName has already been uninstalled by other means."
+} elseif ($key.Count -gt 1){
+    Write-Warning "$key.Count matches found!"
```

Comment:
change the `$key.Count` to `$($key.Count)`, otherwise the script will output wrongly to the user when this is reached.